File: code3-1.py
n, v = map(int, input().split())
a = []

for i in range(n):
    ai = int(input())
    a.append(ai)

exist = False
found_id = -1
for i in range(n):
    if a[i] == v:
        exist = True
        found_id = i

if exist:
    print("Yes")
    print(found_id)
else:
    print("No")

File: code3-1.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    //入力を受ける
    int N, v;
    cin >> N >> v;
    vector<int> a(N);
    for(int i = 0; i < N; ++i) cin >> a[i];

    //線形探索
    bool exist = false;
    for(int i = 0; i < N; ++i) {
        if (a[i] == v) {
            exist = true;
        }
    }

    //結果出力
    if(exist) cout << "Yes" << endl;
    else cout << "No" << endl;
}